### DISCUSSION TITLE
How to mount ADXL (Pico)?

### DISCUSSION THREAD

**Discussion by dev-zetta** [Guilouz/Klipper-Flsun-Speeder-Pad#104] [Q&A] [answered]
Hello, where should I mount the adxl - pico version? On the fan cover?

**Comment by Guilouz** [comment, marked as answer]
https://www.printables.com/model/268463-flsun-v400-adxl345-mount

**Comment by dev-zetta** [comment]
Great, I used this, just wasn't sure if I have to change the axes in the
config in some way. Thank you :)

On Sat, May 20, 2023 at 2:49 PM Cyril Guislain ***@***.***>
wrote:

> Closed #104
> <https://github.com/Guilouz/Klipper-Flsun-Speeder-Pad/discussions/104> as
> resolved.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Guilouz/Klipper-Flsun-Speeder-Pad/discussions/104>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAZBWB7LVKWYAYLDEA3P4NTXHC4TZANCNFSM6AAAAAAYD3OEDQ>
> .
> You are receiving this because you authored the thread.Message ID:
> <Guilouz/Klipper-Flsun-Speeder-Pad/repo-discussions/104/discussion_event/674746
> @github.com>
>


-- 
Have a nice day,

Gabriel
